autogpt/llm_utils.py

```diff
@@ -81,20 +84,28 @@ def create_chat_completion(
     for attempt in range(num_retries):
         backoff = 2 ** (attempt + 2)
         try:
-            if CFG.use_azure:
-                response = openai.ChatCompletion.create(
-                    deployment_id=CFG.get_azure_deployment_id_for_model(model),
-                    model=model,
-                    messages=messages,
-                    temperature=temperature,
-                    max_tokens=max_tokens,
-                )
-            else:
-                response = openai.ChatCompletion.create(
-                    model=model,
-                    messages=messages,
-                    temperature=temperature,
-                    max_tokens=max_tokens,
+            if CFG.llm_provider == "openai":
+                if CFG.use_azure:
+                    response = openai.ChatCompletion.create(
+                        deployment_id=CFG.get_azure_deployment_id_for_model(model),
+                        model=model,
+                        messages=messages,
+                        temperature=temperature,
+                        max_tokens=max_tokens,
+                    )
+                else:
+                    response = openai.ChatCompletion.create(
+                        model=model,
+                        messages=messages,
+                        temperature=temperature,
+                        max_tokens=max_tokens,
+                    )
+            if CFG.llm_provider == "anthropic":
+                response = anthropic_client.completion(
+                    prompt=f"{anthropic.HUMAN_PROMPT} {messages} {anthropic.AI_PROMPT}",
+                    stop_sequences = [anthropic.HUMAN_PROMPT],
+                    model="claude-v1.3",
```

Comment:
Should be safe to use `claude-v1`, which should be kept up to date with the latest-and-greatest.

Reply:
Updated https://github.com/anthony-sarkis/Auto-GPT/commit/b47a9331c19c63fa233dc575eb97cffbae041485